cinema/views.py

```diff
@@ -126,21 +123,43 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "item": {"type": "number"}},
```

Comment:
As I remember, we search only by one title name, so type={"type": "string"} would've been better option.